prow/plugins/plugins.go

```diff
@@ -122,6 +120,14 @@ func (pa *PluginAgent) Load(path string) error {
 	if err := yaml.Unmarshal(b, &np); err != nil {
 		return err
 	}
+	return pa.Set(np)
+}
+
+// Set sets the plugins that are enabled on repos. The input is a map from
```

Comment:
This reads weird, and then in my head I went off on a tangent about how naming things is hard, and I don't know if it really matters.  

But above, `Load` "attempts", because it could err.  Maybe same here? Otherwise LGTM